css-view-transitions-1/Overview.bs

```diff
@@ -183,9 +463,9 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 	This may be confusing, since the elements themselves are not necessarily referring to the same object,
 	but it is a useful model to consider them to be visual states of the same conceptual page entity, that we happen to call "element".
 
-# User-agent keyframes # {#ua-keyframes}
+# User-agent styles # {#ua-styles}
```

Comment:
Although this section doesn't contain all of the UA styles (feels like a lot of them are better off in the pseudo-element section), it gives a heading for readers to see, and the note in this section directs them to other UA styles.